infra-as-code/bicep/modules/logging/logging.parameters.example.json

```diff
@@ -36,6 +39,16 @@
           "Environment": "POC"
       }
     },
+    "parAutomationAccountTags": {
+      "value": {
+          "Environment": "POC"
+      }
+    },
+    "parLogAnalyticsWorkspaceTags": {
```

Comment:
Let's emit/remove the `parAutomationAccountTags` and `parLogAnalyticsWorkspaceTags` and leave them to default